### PR TITLE
added subcomponents and tests for subcomponents

### PR DESCRIPTION
**Pull request by jackzhen1996**

### CODE REVIEW

File: client/components/Reviews/__test__/Review.test.js [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+import Reviews from '../components/Reviews.js';
+import Ratings from '../components/Ratings.js';
+import ReviewsAndRatings from '../index.js';
+
+// set up enzyme's react adapter
+Enzyme.configure({ adapter: new EnzymeAdapter() });
+
+/*
+test('Review Component Exists', () => {
```
Comment (melliottgithub) [inline client/components/Reviews/__test__/Review.test.js:13]: try to erase comments before you commit I did the same thing on a few of my last commits

**Review by melliottgithub — APPROVED** [review]

File: client/GlobalStyles.js [hunk 2]
```diff
@@ -4,7 +4,7 @@ const GlobalStyles = createGlobalStyle`
     body{
         margin:0;
         padding:0;
-        font-size: 62.5%;
+        font-size: 1rem;
```
Comment (melliottgithub) [inline client/GlobalStyles.js:7]: use fosnt size 1 rem on your local styles

**Review by melliottgithub — CHANGES_REQUESTED** [review]

File: client/GlobalStyles.js (hunk 2 above)
Comment (jackzhen1996) [reply client/GlobalStyles.js]: @melliottgithub corrected this and added local style

File: client/components/Reviews/__test__/Review.test.js (hunk 1 above)
Comment (jackzhen1996) [reply client/components/Reviews/__test__/Review.test.js]: @melliottgithub deleted the comments

**Review by melliottgithub — APPROVED** [review]

**Pull request merged**